File: src/main/java/toker/panel/repository/CraftingStationRepository.java
package toker.panel.repository;

import toker.panel.entity.CraftingStation;
import toker.panel.entity.pk.CraftingStationPK;

import java.util.List;

public interface CraftingStationRepository
        extends BaseRepository<CraftingStation, CraftingStationPK> {
    List<CraftingStation> findAllByServerId(Integer serverId);
}

File: src/main/java/toker/panel/repository/CraftingStationRepository.kt
package toker.panel.repository

import toker.panel.entity.CraftingStation
import toker.panel.entity.pk.CraftingStationPK

interface CraftingStationRepository : BaseRepository<CraftingStation, CraftingStationPK> {
    fun findAllByServerId(serverId: Int): List<CraftingStation>
}